Wrappers/Python/cil/optimisation/operators/MatrixOperator.py

```diff
@@ -55,6 +67,21 @@ def direct(self,x, out=None):
             return out
 
     def adjoint(self,x, out=None):
+        r"""Returns :math:`A^{T}x`
+        
+        Parameters
+        ----------
+        x : DataContainer
+            Input data
+        out : DataContainer, optional
+            Output data, default is None
+            
+        Returns
+        -------
+        DataContainer
```

Comment:
can this be a BlockDataContainer?

Reply:
No because BlockDataContainer doesn't have a fill method